azkaban-exec-server/src/main/bash/azkaban-executor-shutdown.sh

```diff
@@ -5,15 +5,30 @@ installdir="$(dirname $0)/.."
 pid=`cat ${installdir}/currentpid`
 port=`cat ${installdir}/executor.port`
 
-echo "Killing Executor. [pid: $pid, port: $port]"
-
-kill ${pid}
-if [ $? -ne 0 ]; then
-    echo "Error: Shutdown failed"
-    exit 1;
+maxtry=5
+if [ -z $pid ]; then
```

Comment:
@li-afaris i think 'set -o errexit' might cause undesirable results. For example, killing a non-existing process will exit the shell with error code which might be captured by deployment script treating it as a deployment failure, but in this case we actually want to proceed deployment.